components/net/Cargo.toml

```diff
@@ -15,9 +15,9 @@ brotli = "1.0.6"
 cookie = "0.6"
 devtools_traits = {path = "../devtools_traits"}
 flate2 = "0.2.0"
-hyper = "0.10"
+hyper = "0.10.8"
 hyper_serde = "0.6"
-hyper-openssl = "0.2.2"
+hyper-openssl = "0.2.6"
```

Comment:
Same reason, though I ended up not using the `lock` function I introduced in `0.2.6`, given we don't need to look at the cipher yet.